components/Footer.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <footer class="w-full bg-gray-100 pt-14">
+    <div class="w-[1024px] mx-auto">
+      <div class="grid grid-cols-4 mb-16 w-full">
+        <div>
+          <p class="mb-5">Follow us</p>
+          <p class="font-3 mb-5 font-bold">nn-nn-nn</p>
+          <ul class="flex">
+            <li class="mr-6">
+              <a href="#">
+                <svg
```

Comment:
what you think to set svgs in assets and import them as `<img :src="svg" />`? Component will look kinda more clear 👌 but I think it's up to you, it's not the main purpose of the project 😄

Reply:
I decided to skip it for exactly that reason